civihr_employee_portal/views/views_export/notifications/mydetails_notifications__emergency_contacts.inc

```diff
@@ -0,0 +1,202 @@
+<?php
+
+$view = new view();
```

Comment:
I wasn't clear just from reading the PR description, why are we creating all these new views? Is it just to generate the email content? If that's the case then why not just use smarty templates if we already have all the data required to generate them?

Reply:
@mickadoo new views serves for obtaining results before and after, with that information differences are calculated. Those differences are sent in the notification email. Hence new views are created with 2 purposes: for generating differences and generating the notifications.